src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/MemoryAddress.java

```diff
@@ -189,12 +205,14 @@ default MemorySegment asSegmentRestricted(long bytesSize) {
     int hashCode();
 
     /**
-     * The off-heap memory address instance modelling the {@code NULL} address.
+     * The native memory address instance modelling the {@code NULL} address, associated
+     * with the {@link ResourceScope#globalScope() global} resource scope.
```

Comment:
`{@linkplain }` ?

Reply:
thanks - I'll do a pass - I think I probably got all of them wrong